notebook.ipynb

```diff
@@ -86,43 +86,2150 @@
GqtXr57Wrl1rOr5+/bomTZqkVatW6dq1a3r22Wc1depUVa5cudCa169f1/PPP6+jR49q06ZN8vf3L7Y+g8FQ5PjYsWM1ZMgQSVL37t21e/dus/kXXnhBc+bMKdkmAAAAAAAAAAAA4KExGo0asX6rVh87oc19X5GHY3mzeZtSVgqsXEnJyclm4ykpKapSpcodrZWWlqbu3burdu3amjlzpiwtLUt8roODg2ndgwcP6s0335QkhYeHKzQ01Cy2devWmjhxotq2bXtH9QEAAAAAAAAAAAAAAODxRqMYAADwWNmRelpDGtZVA0Ml3SgwasKWnXo+brkOvt5XZW2sTXHtvd31zrKVpmNra2uzPBMmTNDmzZs1e/ZsVahQQW+99Zb69OmjjRs3ysrKyix28uTJcnV11dGjR29bX1JSktnx119/rYiICHXq1MlsvGfPnnrjjTdMx3Z2dre/eAAAAAAAAAAAADx0w9Zt1ZJDx7Q85AXZ29jobM4VSVJ5OxuV/t/foEY2C1TPFV+qcePGatq0qbZt26bNmzfriy++MOXJyMhQRkaGUlNTJUnHjh1T2bJlZTAY5OjoqLNnz+qVV15R5cqVNW7cOP3yyy+mc11cXEyvg4KCNGbMGHXs2FGS9OWXX6pixYoyGAw6duyYxo8frw4dOqhFixamc39//m8MBoOqVq16fzcLAAAAAAAAAAAAAAAAjxSNYgAAwGNlba9uZsexL7WX4b2PtP/nc2ru/n9PZLSxsiryhldJys7O1pIlS/T+++8rKChIkhQTE6OGDRtqx44datmypSl2w4YN+uabbxQbG6utW7fetr4/rrlp0yY1bdpU1apVMxu3s7Mrtj4AAAAAAAAAAAA8Pj5OPChJarNwmdn4vBfbq3c9P0nSS7V89M47LRUTE6Px48fL09NTsbGxatSokSk+Li5OM2bMMB137dpVkjRjxgwFBwfrm2++0YkTJ3TixAkFBgaarZWWlmZ6nZycrOzsbNNxRkaG3nrrLWVmZsrFxUXdu3fXiBEj7s/Fo5A9ORb3niQ7/X8v7j5XY3vjvdcBAAAAAAAAAAAAAACeODSKAQAAj7Wsa9clSY6l7czGt6eeUUBAgBwcHNSkSRONHj1aTk5OkqTvv/9e+fn5pqcoSpKrq6tq1KihxMREU6OY8+fPa+DAgZo3b55Kly59x7WdP39eW7Zs0axZswrNrVy5UitWrJCzs7Oee+45jRw5UuXKlbvjNQAAAAAAAAAAAPBg5U0cWaK4Hj16qEePHsXOR0REKCIiotj54OBgjRgxQunp6TIai28C8vumMZL06quv6tVXXy1RjcXlAAAAAAAAAAAAAAAAwJOBRjEAAOCxZTQa9eamb9SsqkH+lZxM4+19PNTNr7rsXx2qn376Se+9955eeeUVbdiwQba2tjp//rxsbGz01FNPmeVzdnZWRkaGKfeIESM0ePBg1alTRz/99NMd17ds2TKVK1dOHTt2NBvv0qWL3Nzc5OLiouPHjysqKkpHjx7VkiVL7nwTAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEA0igEAAI+x4eu36vC5TH3dP9hs/BX/GpKk8zVrqmbNmqpTp46eeeYZbdmyRZ06dSo2n9FolIWFhSRp/vz5ysnJ0ZgxY0zNY+7UkiVL1KVLF9nZ2ZmN9+zZ0/S6Zs2a8vDwUMeOHXXo0CHVrl37rtYCAAAAAAAAAAAAAAAAAAAAAAAAAAAA8NdGoxgAAPBYGrF+q9YeT9aWfsGqUt7+lrGVKlWSwWDQyZMnJUnOzs7Ky8vTpUuX9NRTT5niMjMzFRgYKEnatWuX9u/fL1tbW7NcnTp1UpcuXfT+++/fcs3vvvtOycnJmjNnzm2vpXbt2rK2tlZKSgqNYgAAAAAAwH2zJ8fi3pNkp//vxd3namxvvPc6AAAAAAAAAAAAAAAAAAAAANyW5aMuAAAA4PeMRqOGr9uiVT/8qE19XpaHY/nbnnPhwgWlp6fLxcVFkhQQECBra2tt377dFHPu3DkdP37c1Chm0qRJ+uqrr3TgwAFt3rxZcXFxkqQ5c+Zo9OjRt10zPj5eAQEB8vPzu23s8ePHlZ+fr0qVKt02FgAAAAAAAAAAAAAAAAAAAAAAAAAAAACKUupRFwAAAPB7w9Zt1ZJDx7Q85AXZ29jobM4VSVJ5OxuVtrbW5et5mrRtt7r4+sjm9GmdPn1a77zzjhwdHdWxY0dJkoODg3r06KG3335bjo6OcnR01Ntvv62aNWuqefPmkiSDwSALCws9/fTTSk9PV5kyZSRJ1apVU+XKlU31BAUFacyYMabckpSTk6O1a9dq/PjxhepPTU3VypUr1apVK1WoUEH//e9/9fbbb8vf318NGzZ8YPsGAAAAAAAAAAAAAAAAAAAAAAAAAAAA4MlGoxgAAPBY+TjxoCSpzcJlZuPzXmyv3vX8ZGVpocMZmfrs4FFd+vdyubi4qGnTppozZ47KlStnip84caJKlSqlwYMH69q1a3r22Wc1c+ZMWVlZ3VE9ycnJys7ONhtbvXq1jEajXnrppULx1tbW2rlzp+bNm6erV6+qcuXKat26tf7xj3/c8doAAAAAAAAAAAAAAAAAAAAAAAAAAAAA8BsaxQAAgMdK3sSRt5wvbW2tdb26SZLOD4woNs7Ozk6TJ0/W5MmTS7Sum5ub0tLSCo0XNRYWFqawsLAi8xgMBi1fvrxEawIAAAAAAAAAAAAAAAAAAAAAAAAAAABASdEoBgAAAAAAAAAAAAAAALiNPTkW954kO/1/L+4+V2N7473XAQAAAAAAAAAAAAAAgD8ly0ddAAAAAAAAAAAAAAAAAAAAAAAAAAAAuDvTduxVk7mLVGFqjAzvzlG3+NU6nnnBLObtr7+Vf8wCeXt7y9fXV8HBwdq/f3+R+YxGo8LCwmQwGLRx40azuT59+qhq1ary8PBQvXr1NHToUJ09e/aW9Z0/f14jRoxQ/fr15eXlpZ49eyolJcUsJiMjQ0OHDlXd
```

Comment:
on the top table there are no values along the x axis

Reply:
The two graphs share the same x-axis